Added Wattage, Lenspect, Identities, Blurble, Atomix, GNOME Sudoku (#373)
myfreeweb/awesome-gtk: README.md

- [Collision](https://apps.gnome.org/app/dev.geopjr.Collision) - Tool to generate, compare and verify hashes `#crystal` `#gtk4` `#libadwaita` `#gnome`.
- [GtkHash](https://gtkhash.org) - Desktop utility for computing message digests or checksums `#c` `#gtk3`.
- [Kapitano](https://flathub.org/apps/page.codeberg.zynequ.Kapitano) - User-friendly antivirus tool and malware scanner for Linux (malware, viruses, and threats) based on ClamAV `#python` `#gtk4` `#libadwaita`.

> - [Lenspect](https://flathub.org/en/apps/io.github.vmkspv.lenspect) - Security threat and malware scanner for files `#python` `#gtk4` `#libadwaita`.

Reviewer: ```suggestion
- [Lenspect](https://flathub.org/apps/io.github.vmkspv.lenspect) - Security threat and malware scanner for files `#python` `#gtk4` `#libadwaita`.
```

- [Authenticator](https://apps.gnome.org/Authenticator) - Two-factor authentication codes generator `#rust` `#gtk4` `#libadwaita` `#gnome`.
- [Goldwarden](https://flathub.org/apps/com.quexten.Goldwarden) - Bitwarden compatible desktop password manager `#go` `#gtk4` `#libadwaita`.
- [Gonepass](https://github.com/jbreams/gonepass) - 1Password vault reader `#c++` `#gtk3`.

> - [Identities](https://flathub.org/en/apps/one.k8ie.Identities) - `pass` (password store) client for the GNOME desktop `#python` `#gtk4` `#libadwaita`.

Reviewer: ```suggestion
- [Identities](https://flathub.org/apps/one.k8ie.Identities) - `pass` (password store) client for the GNOME desktop `#python` `#gtk4` `#libadwaita`.
```

> - [Identities](https://flathub.org/en/apps/one.k8ie.Identities) - Password store client (pass fronted) for the GNOME desktop `#python` `#gtk4` `#libadwaita`.

Reviewer: ```suggestion
- [Identities](https://flathub.org/en/apps/one.k8ie.Identities) - `pass` (password store) client for the GNOME desktop `#python` `#gtk4` `#libadwaita`.
```

- [Touché](https://flathub.org/apps/com.github.joseexposito.touche) - User interface to configure touchpad and touchscreen multi-touch gestures `#gjs` `#javascript` `#gtk4` `#libadwaita`.
- [Tuner](https://flathub.org/apps/org.altlinux.Tuner) - Extensible control center for additional GNOME desktop system settings, components, applications `#vala` `#gtk4` `#libadwaita`.
- [Warehouse](https://github.com/flattool/warehouse) - Toolbox for managing Flatpak packages (view info, manage user data, pin versions, change version, install/uninstall, etc.) `#python` `#gtk4` `#libadwaita`.

> - [Wattage](https://flathub.org/en/apps/io.github.v81d.Wattage) - Monitoring application of the health and status of power devices, e.g. battery capacity, energy metrics, and device information `#vala` `#gtk4` `#libadwaita`.

Reviewer: ```suggestion
- [Wattage](https://flathub.org/apps/io.github.v81d.Wattage) - Monitoring application of the health and status of power devices, e.g. battery capacity, energy metrics, and device information `#vala` `#gtk4` `#libadwaita`.
```

### Puzzles and Quizzes

> - [Atomix](https://gitlab.gnome.org/GNOME/atomix) - Puzzle game in which the user has to build full molecules `#c` `#gtk3`.

Reviewer: ```suggestion
- [Atomix](https://flathub.org/en/apps/org.gnome.atomix) - Puzzle game in which the player has to build full molecules `#c` `#gtk3`.
```

> - [Blurble](https://flathub.org/en/apps/app.drey.Blurble) - Word guessing game `#vala` `#gtk4` `#libadwaita`.

Reviewer: ```suggestion
- [Blurble](https://flathub.org/apps/app.drey.Blurble) - Word guessing game `#vala` `#gtk4` `#libadwaita`.
```

- [Egghead](https://flathub.org/apps/io.github.josephmawa.Egghead) - Knowledge and trivia quiz, including categories like "Science and Nature", "Geography", and "Politics" `#gjs` `#javascript` `#gtk4` `#libadwaita`.

> - [GNOME Sudoku](https://gitlab.gnome.org/GNOME/gnome-sudoku) - Digitalized version of Sudoku, the classic Japanese puzzle game `#vala` `#gtk4` `#libadwaita`.

Reviewer: ```suggestion
- [GNOME Sudoku](https://flathub.org/apps/org.gnome.Sudoku) - Digitalized version of Sudoku, the classic Japanese puzzle game `#vala` `#gtk4` `#libadwaita`.
```
